arrow/test/test_arrow.py

```diff
@@ -100,28 +101,33 @@ def load_state(filename):
     duration = 1
 
     # semi-quantitative rate constants
-    rates = np.full(n_reactions, 1000)
+    rates = np.full(n_reactions, 1000.0)
 
-    return stoichiometric_matrix, rates, initial_state, final_state
+    return (stoichiometric_matrix, rates, initial_state, final_state)
 
 def complexation_test(make_system):
-    stoichiometric_matrix, rates, initial_state, final_state = load_complexation()
+    stoichiometric_matrix, rates, initial_state, final_state = load_complexation(prefix='beginning')
     duration = 1
 
-    system = make_system(stoichiometric_matrix, rates)
+    system = make_system(stoichiometric_matrix, rates, random_seed=np.random.randint(2**31))
```

Comment:
Does this seed the random number generator? It's good to have repeatable unit tests.

Reply:
Yep, that was for my own purposes during testing but I can take it out now.